src/prototype_creep_harvest.js

```diff
@@ -29,7 +29,7 @@ Creep.prototype.spawnCarry = function() {
     resourceAtPosition += _.sum(container.store);
   }
 
-  if (resourceAtPosition > parts.carryParts.carry * CARRY_CAPACITY || (this.memory.routing.type === 'commodity' && this.store.getFreeCapacity === 0)) {
+  if (resourceAtPosition > parts.carryParts.carry * CARRY_CAPACITY || (this.memory.routing.type === 'commodity' && this.store.getFreeCapacity() === 0)) {
```

Comment:
this should be a function call, yes ?
changed `this.store.getFreeCapacity` to `this.store.getFreeCapacity()`

Reply:
yes, correct, thx, good catch